src/coreComponents/fileIO/python/PyHistoryCollection.cpp

```diff
@@ -19,7 +17,7 @@
 
 namespace geosx
 {
-
+using namespace dataRepository;
```

Comment:
Why do we need to add this namespace ?

Reply:
It was to access the constant geosx::dataRepository::keys::domain, but its is no longer required in the latest commit (I have removed this line)